### PR TITLE
Feat:Android app development  success story

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
**##** Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Description:**
1.Fix top marginof  landing section text in 4k device.
2. Add links in case study section to specific portfolios.
3. Change bold text in blog section.
4. Add loop in development section.
5. FAQ section: Make arrow interchangable.
6. BLog section: Remove title and add image, remove opacity on hover.
7. Fix CTA height.
8. Fix Vertical Soacing.
9. Create Success Story section . 

**Mobile Performance:**
image
**Desktop Performance:**
image

Desktop Image:
image

Mobile Image:
image

Video:

https://github.com/canopas/canopas-website/assets/98794636/a5eedbf4-e318-4d92-b763-1b5455e68443

### CODE REVIEW

File: vue-frontend/src/components/android-app-development/SuccessStorySection.vue [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+            :spaceBetween="20"
+            :direction="'vertical'"
+            :autoplay="{
+              delay: 1000,
+              disableOnInteraction: false,
+            }"
+            :loopedSlides="100"
+            :speed="8000"
+            :loop="true"
+            class="swiper-container tw-h-[700px]"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/SuccessStorySection.vue:29]: success stories section working fine in auto , but when we hover other user image it not working good see.

https://github.com/canopas/canopas-website/assets/115449373/a7be15a8-5c04-4166-a2a6-d237464c34ef

File: vue-frontend/src/components/android-app-development/SuccessStorySection.vue [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import SwiperCore, { Autoplay } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import usericon from "@/assets/images/andriod-app-development/icon.webp";
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/android-app-development/SuccessStorySection.vue:205]: its not looking like figma's icon

Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/SuccessStorySection.vue:205]: digita told me to prefer link website. as figma don't support many animation

Comment (cp-jagruti-a) [reply vue-frontend/src/components/android-app-development/SuccessStorySection.vue:205]: i am talking about icon

File: vue-frontend/src/components/android-app-development/SuccessStorySection.vue [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+              :direction="'vertical'"
+              :autoplay="{
+                delay: 4000,
+                disableOnInteraction: false,
+              }"
+              :loop="true"
+              :loopedSlides="50"
+              class="swiper-container tw-h-[600px] lg:tw-h-[700px]"
+              @swiper="setSwiperRef"
+              @slideChange="onSlideChange"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/android-app-development/SuccessStorySection.vue:102]: its working like this when we click below it refer attachment

https://github.com/canopas/canopas-website/assets/85920585/d78a874d-ba14-471e-857f-6de1a6c2dd51

File: vue-frontend/src/components/android-app-development/SuccessStorySection.vue [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+              :centeredSlidesBounds="true"
+              :spaceBetween="10"
+              :slideToClickedSlide="true"
+              :direction="'vertical'"
+              :autoplay="{
+                delay: 4000,
+                disableOnInteraction: false,
+              }"
+              :loop="true"
+              :loopedSlides="50"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/SuccessStorySection.vue:99]: only set scroll on click in aythor profile and auto.
can you able to remove this scrolling ?

https://github.com/canopas/canopas-website/assets/115449373/dff230d5-f02b-4f40-b870-93560a29d20b

**Review by cp-mansi-d — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**